projects/plugins/jetpack/changelog/remove-imploding-css-code

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: enhancement
+
+Removed the CSS imploding functionality.
```

Comment:
"CSS imploding" may not be clear to most of our customers. Could we phrase this differently maybe, highlighting that we're making changes to how assets are loaded on each page, only when necessary?